src/layouts/Workspace.jsx

```diff
@@ -36,15 +38,14 @@ const Workspace = ({ match, location }) => {
             // create option for contact-us page
           }
         }
-
-        try { 
-          const collectionsResp = await axios.get(`${BACKEND_URL}/sites/${siteName}/collections`);
-          if (_isMounted) setCollections(collectionsResp.data?.collections)
-        } catch (e) {
-          setCollections(undefined)
-          console.log(e)
+        try {
+            const collectionsResp = await axios.get(`${BACKEND_URL}/sites/${siteName}/collections`);
+            if (_isMounted) setCollections(collectionsResp.data?.collections)
+        } catch (err) {
+            setCollections(undefined)
+            console.log(err)
+            if (err?.response?.status === 404) setShouldRedirect(true)
```

Comment:
What is the rationale for redirecting to `Not Found` only if we cannot find collections, but not when we cannot find unlinked pages?

Reply:
No longer necessary after the rework in https://github.com/isomerpages/isomercms-frontend/pull/280/commits/29101878097a4e7a4c7a80df5dec76b18b9cb6e0